pkgs/development/python-modules/pandantic/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, multiprocess
+, pandas
+, pandas-stubs
+, poetry-core
+, pydantic
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "pandantic";
+  version = "0.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchFromGitHub {
+    owner = "wesselhuising";
+    repo = "pandantic";
+    rev = "f0d749459876a17bd5db53a69ffdfe3bbfc197f5";
+    hash = "sha256-RXonAk9iy+SFvYtkI2AFIZBogIUDmQfUwKc1Da13viM=";
+  };
+
+  nativeBuildInputs = [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = [
+    multiprocess
+    pandas
+    pandas-stubs
+    pydantic
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [
+    "pandantic"
+  ];
+
+  disabledTests  = [
+    "test_read_with_lxml_dtype_backend"
```

Comment:
For the future it will help if there is a comments about the failure or a link to the upstream issue.